v2/src/lib.rs

```diff
@@ -356,59 +410,46 @@ struct InfluxDbStorage {
 }
 
 impl InfluxDbStorage {
+    fn get_db_name(&self) -> ZResult<String> {
+        let db = match self.config.volume_cfg.get(PROP_STORAGE_DB) {
+            Some(serde_json::Value::String(s)) => s.to_string(),
+            _ => bail!("no db was found"),
+        };
+        Ok(db)
+    }
```

Comment:
Is it possible to have a `InfluxDbStorage` without a name? The error `"no db was found"` is making me think that no. Hence, I'm wondering if we cannot change the `InfluxDbStorage` structure and add a field `db_name` (or similar name) there and set that field in the function `InfluxDbVolume::create_storage` — as far as I understood, the name is known and set at that point.

Reply:
So I have implemented the changes with respect to `get_db_name` 
but if I am not mistaken we can apply config changes to running plugins, 
would there be a case where we would change the config such that we change DB name at runtime ?
I personally don't like the idea of it, but is this a use case we would want to support ?

Reply:
@Mallets any idea w.r.t. changing the plugin configuration and changing the database name?

Reply:
@Charles-Schleich can you check wether this is supported in all backends? We should try to keep this kind of functionality homogeneous as much as possible across all the backends.

Reply:
Looks like none of the storage back ends support this use case.
Only plugin storage manager.  Resolving this and ready to merge.